src/config/2018/picks.hockey.js

```diff
@@ -0,0 +1,30 @@
+module.exports = `--round 1--
```

Comment:
Originally, this was just a `.txt` (or rather, my newly-invented, `.hockey` file), but I had to make it JS for use with `require` on the client. I was using `fs` to load the file, but that does not work in the browser and I don't care to polyfill such things for the browser.